src/files-and-videos/files-page/FilesPage.test.jsx

```diff
@@ -223,10 +232,10 @@ describe('FilesAndUploads', () => {
           await waitFor(() => {
             addFilesButton = screen.getByLabelText('file-input');
           });
```

Comment:
This isn't part of your code but here it could be replaced by:

addFilesButton = await screen.getByLabel...